### PR TITLE
update DE locale from sublime text builtin Default package

### PR DESCRIPTION
**Pull request by Henry2o1o**

### CODE REVIEW

**Comment by rexdf** [comment]
Thanks for your pull request.

You can first copy all from EN folder to jp folder, and translate the caption field.

File: DE/Context.sublime-menu [hunk 1]
```diff
@@ -1 +1,16 @@
...
+[
+    { "command": "open_context_url" },
+    { "command": "context_goto_definition", "caption": "Goto Definition" },
+    { "caption": "-", "id": "clipboard" },
+    { "command": "Kopieren" },
+    { "command": "Ausschneiden" },
+    { "command": "Einfügen" },
+    { "caption": "-", "id": "selection" },
+    { "command": "select_all" },
+    { "caption": "-", "id": "Datei" },
```
Comment (rexdf) [inline DE/Context.sublime-menu]: you should only translate caption.   'id' is used for python program.

Comment (rexdf) [reply DE/Context.sublime-menu]: `'command'` and `'id'` are used as programming identifiers

Comment (rexdf) [reply DE/Context.sublime-menu]: you should add `caption` after `command`. Now I have updated the repo

Comment (Henry2o1o) [reply DE/Context.sublime-menu]: ok

**Pull request closed without merging**